salt/edx/mitx_etl.sls

```diff
@@ -0,0 +1,57 @@
+install_python3_pip:
+  pkg.installed:
+    - pkgs:
+        - python3
+        - python3-pip
+    - refresh: True
+    - require:
+      - pkg: install_python_3
+
+create_mitx_directory:
+  file.directory:
+    - name: /mitx
+
+clone_mitx_etl_repo:
+  git.latest:
+    - name: https://github.com/mitodl/odl-etl
+    - target: /mitx
+    - force_clone: True
+    - require:
+      - pkg: install_python_3
```

Comment:
This should require the `create_mitx_directory` step

Reply:
Changed